parse/parse.d.ts

```diff
@@ -835,8 +848,8 @@ declare namespace Parse {
         }
 
         interface FunctionResponse {
-            success?: (response: HttpResponse) => void;
-            error?: (response: HttpResponse) => void;
+            success?: (response: any) => void;
```

Comment:
From the [docs](https://parse.com/docs/js/api/classes/Parse.Cloud.FunctionResponse.html):

> If success is called, will return a successful response with the optional argument to the caller.

As a matter of fact, one can pass basically any object to the caller, for example [the official docs call `success` with a `float` and `error` with a `string`](https://parse.com/docs/cloudcode/guide#cloud-code-cloud-functions).